erpnext/projects/doctype/project/project.py

```diff
@@ -1273,7 +1290,7 @@ def set_items_and_totals_html_onload(self, sales_data, consumables_data):
 
 	def get_project_sales_data(self, get_sales_invoice=True):
 		sales_data = frappe._dict()
-		sales_data.material_items, sales_data.part_items, sales_data.lubricant_items = get_material_items(self,
+		sales_data.material_items, sales_data.part_items, sales_data.lubricant_items, sales_data.consumable_items, sales_data.total_discount_amount = get_material_items(self,
```

Comment:
wrong location for `total_discount_amount`, it should be in `sales_data.totals.total_discount_amount`